src/AxisKeys.jl

```diff
@@ -1,7 +1,7 @@
 module AxisKeys
 
 include("struct.jl")
-export KeyedArray, axiskeys
+export KeyedArray, axiskeys, named_axiskeys
```

Comment:
or spell it `namedaxiskeys`? Should it be exported?

Reply:
Name seems OK, it follows NamedDims's & is getting a bit long as one word. Could also be `named_keys` to shorten it. 

Would you mind moving it to the end of file names.jl?

Reply:
done.